Firmware/code.py

```diff
@@ -5,72 +5,36 @@
 from adafruit_hid.keyboard import Keyboard
 from adafruit_hid.keycode import Keycode
 
-#Setting pin varables
-Screen_1_Btn_Pin = board.GP1
-Screen_2_Btn_Pin = board.GP2
-Screen_3_Btn_Pin = board.GP3
-Screen_4_Btn_Pin = board.GP4
-Censor_Btn_Pin = board.GP15
+class BtnControl:
+    def __init__(self, name, pin, key):
+        self.name=name
+        self.btn=digitalio.DigitalInOut(pin)
+        self.btn.direction=digitalio.Direction.INPUT
+        self.btn.pull=digitalio.Pull.DOWN
+        self.key=key
+        self.last_value=False
+
+    def update(self):

```

Comment:
Feels like this function would set self.last_value and never set it back to None, which seems like it would mean you could never press that button again

Reply:
idk what the actual values are (numbers? bool?). Considering Flux used "if button.value" to detect presses, it must mean the value is falsy when not pressed and truthy when pressed. thus line 25 `self.last_value=self.btn.value` will be set to falsy (whatever that falsy value turns out to be) if the button is not pressed, allowing it to be pressed again.